.prettierignore

```diff
@@ -1,4 +1,3 @@
 .yarn
 **/*/storybook-static/*
-*.yml
```

Comment:
This change and the associated formatting changes don't seem to have anything to do with this PR.  Could you please move them to a separate PR?

Reply:
I found that GitHub workflows files is not formatted.

This change is needed for linting of the YAML files.

Yes, I can move them to a separate PR.

Reply:
Fixed.